File: Edit-Profile-Service/src/main/kotlin/com/kinok0/editprofileservice/repository/TaskRepository.java
package com.kinok0.editprofileservice.repository;

import com.kinok0.editprofileservice.entity.TaskEntity;
import org.springframework.data.repository.CrudRepository;

import java.util.List;
import java.util.UUID;

public interface TaskRepository extends CrudRepository<TaskEntity, UUID> {

    @Override
    List<TaskEntity> findAll();
}

File: Profile-Service/src/main/kotlin/com/kinok0/profileservice/repository/TaskRepository.kt
package com.kinok0.profileservice.repository

import com.kinok0.profileservice.entity.TaskEntity
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import java.util.UUID

@Repository
interface TaskRepository : CrudRepository<TaskEntity, UUID> {
    fun getTaskEntityById(id: UUID) : TaskEntity
}